tests/scr-readingsystem-support_iframe/OPS/iframe_content.xhtml

```diff
@@ -0,0 +1,53 @@
+<html xmlns="http://www.w3.org/1999/xhtml" xmlns:epub="http://www.idpf.org/2007/ops" xml:lang="en">
+  <head>
+    <title>epubReadingSystem object support in an iframe</title>
+    <script>
+      /**
+       * A simple change that proves that scripting is enabled.
+       * 
+       * Note that the script is more complicated than would be necessary in HTML, due to the fact that the innerHTML method does not
+       * seem to work in xhtml (only in html) :-(
+       */ 
+      function test_ers() {
+        try {
+          const ers = navigator.epubReadingSystem;
+          // For code testing
+          // const ers = {
+          //   name : "Test ERS setup",
+          //   version: "0.0.0"
+          // };
+          if (ers !== undefined) {
+            const div_pass = document.getElementById('pass');
+            const div_fail = document.getElementById('fail');
+            const ers_name = document.getElementById('ers_name');
+            const ers_version = document.getElementById('ers_version');
+
+            ers_name.innerText = ers.name;
+            ers_version.innerText = ers.version;
+            div_fail.style.display = 'none';
+            div_pass.style.display = 'block';
+          }
+        } catch (e) {
+          // Let the original, negative display stay as is. 
+        }
+      }
+
+      window.addEventListener('load', test_ers);
+    </script>
+  </head>
+  <body>
+    <div id="fail" style="display: block">
+      <p>The Reading System does not implement the <code>epubReadingSystem</code> object within an <code>iframe</code>. Test fails.</p>
+    </div>
+
+    <div id="pass" style="display: none">
+      <p>The Reading System implements the <code>epubReadingSystem</code> object within an <code>iframe</code>, with the following properties:</p>
+      <ul>
+        <li>Reading System Name: <span id="ers_name"></span></li>
+        <li>Reading System Version: <span id="ers_version"></span></li>
```

Comment:
Same question as previous test: what if ERS is defined but one or both of these properties is not?

Reply:
Right, and I have copied the same text as in https://github.com/w3c/epub-tests/pull/100#discussion_r783149590